hw/ip/entropy_src/data/entropy_src_sec_cm_testplan.hjson

```diff
@@ -24,70 +24,127 @@
+      // The DUT isn't expected to signal an alert though in case of invalid encodings for these two signals.
       name: sec_cm_intersig_mubi
-      desc: "Verify the countermeasure(s) INTERSIG.MUBI."
+      desc: '''
+            Verify the countermeasure(s) INTERSIG.MUBI.
+            Verify that unless the otp_en_entropy_src_fw_read or otp_en_entropy_src_fw_over input signals are equal to MuBi8True the DUT doesn't allow reading entropy from the ENTROPY_DATA register or from the FW_OV_RD_DATA register, respectively.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_rng"]
     }
     {
       name: sec_cm_main_sm_fsm_sparse
-      desc: "Verify the countermeasure(s) MAIN_SM.FSM.SPARSE."
+      desc: '''
+            Verify the countermeasure(s) MAIN_SM.FSM.SPARSE.
+            The entropy_src_functional_errors test verifies that if the FSM state is forced to an illegal state encoding this is reported in the ERR_CODE register.
+            It seems that it doesn't check whether the DUT actually triggers a fatal alert.
+            The alert connection itself is verified through automated FPV.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_sec_cm", "entropy_src_functional_errors"]
     }
     {
       name: sec_cm_ack_sm_fsm_sparse
-      desc: "Verify the countermeasure(s) ACK_SM.FSM.SPARSE."
+      desc: '''
+            Verify the countermeasure(s) ACK_SM.FSM.SPARSE.
+            The entropy_src_functional_errors test verifies that if the FSM state is forced to an illegal state encoding this is reported in the ERR_CODE register.
+            It seems that it doesn't check whether the DUT actually triggers a fatal alert.
+            The alert connection itself is verified through automated FPV.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_sec_cm", "entropy_src_functional_errors"]
     }
     {
       name: sec_cm_rng_bkgn_chk
-      desc: "Verify the countermeasure(s) RNG.BKGN_CHK."
+      desc: '''
+            Verify the countermeasure(s) RNG.BKGN_CHK.
+            Verify the different background health checks with different, randomized threshold values.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_rng"]
     }
     {
       name: sec_cm_ctr_redun
-      desc: "Verify the countermeasure(s) CTR.REDUN."
+      desc: '''
+            Verify the countermeasure(s) CTR.REDUN.
+            The entropy_src_functional_errors test verifies that if there is any mismatch in the redundant counters this is reported in the ERR_CODE register.
+            It seems that it doesn't check whether the DUT actually triggers a fatal alert.
+            The alert connection itself is verified through automated FPV.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_sec_cm", "entropy_src_functional_errors"]
     }
     {
+      // TODO: It seems that this is currently not being verified.
```

Comment:
It would be good to refine this TODO to just encapsulate what is yet undone.

Reply:
We have another issue to track this. We will start working on this shortly.